src/app/table/table.component.html

```diff
@@ -0,0 +1,34 @@
+<section class="table-wrap">
+  <table *ngIf="users.length; else emptyList">
+    <caption>Tabela użytkowników</caption>
+    <thead>
+      <tr>
+        <th scope="col">Id</th>
+        <th scope="col">Nazwa</th>
+        <th scope="col">Stanowisko</th>
+        <th scope="col">Data utworzenia</th>
+        <th scope="col">Data ostatniej aktywności</th>
+        <th scope="col">Akcje</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr *ngFor="let user of users" data-id="{{ user.id }}" [appHighlight]="'lightgreen'" appHighlight2 #trElement
+        (click)="emitTrElement(trElement)">
```

Comment:
co się stanie jeśli w `(click)="emitTrElement(trElement)"` użyjesz `user` w miejsce `trElement`? Czy będzie to pomocne?

Reply:
czy da się z tego zrobić osobny komponent tak aby zachować go do innych tabel?